### DISCUSSION TITLE
T1 overhaul

### DISCUSSION THREAD

**Discussion by AmericanSniper** [Team-QUIET/QUIET-Community-Edition#179] [Ideas]
I overcomplicated my approach to T1 land. I thing I have found the best formula (XD), but I realize I need general help, advice, and other corrections, as well as a solid plan. I want to avoid discovering midway through that the idea is flawed, and I don't want to make arbitrary changes. I need a more scientific perspective.
The T1 Regulars must be balanced with each other to avoid any imbalances in common aspects. For example, I prefer using FAF stats for the 4 vanilla T1 Regulars, but slightly scaled up to better fit the LOUD space and scale.

Aurora: Lowest HP, Slow, Longest Range, Average Cost,
Mantis: Average HP, Fastest, Below Average Range, Average Cost,
Striker: Good HP, Slowest, Average Range, Average Cost.
Thaam: Higher HP, Slow, Above Average Range, Higher Cost.

But I still prefer to maintain that diversity. It’s not completely based on FAF; it’s mostly my own vision of the game. If one unit underperforms and another one overperforms, they can be calibrated to usually result in a 45-55% balance in 50 vs. 50 fights. The additional T1 Regulars are meant to compensate for the weaknesses of the core units, but they should be less effective by comparison, as I want vanilla units to remain the core of the gameplay. Their stats will be adjusted to match the common T1 Regulars

Hervour: Thaam Copy with slightly lower hp and speed.
Bonfire: Mantis Copy with more energy cost and slightly lower speed.
Mite: Striker Copy with slightly lower hp and higher energy cost.
Yenshavoh: Aurora Copy with slightly lower range and higher hp.

This approach would maintain diversity and compensate for the weaker sides of factions, while providing an alternative with a slight penalty in effectiveness. This means that factions specializing in speed will remain the best, even when other factions have alternative units attempting to fulfill the same role. 
T1 Land Scouts: Radar range: 2.08 km, vision: 540 m, larger hitbox, slower speed, with HP comparable to a LAB, and cost equivalent to an average T1 tank with higher energy consumption. No stealth.
The goal may seem unusual, but I don't want scouts to be spammy units. Instead, they should be expensive backline observers. Given the large scale of the game, players often have no idea what’s happening right under their nose.

LABs will be the fastest T1 units, generally costing half of an average T1 tank, with half the HP, damage, and range, but with a larger hitbox.

Mobile AA will be a copy of the FAF version, but with a 960 m radar (the old scout radar). After all, why shouldn’t AA have radar? A short-range radar would make sense.

Artillery should be less cheap, costing more but dealing more damage, while keeping the same HP to maintain balance with LABs. It should have slightly more range than any T1 PD or any Commander’s T1 gun. Overall, the cost will be slightly higher than T1 regulars, with slightly lower HP than the Aurora tank, but with enough damage to quickly destroy PD. The rate of fire will be lower, as it doesn't need high RoF for structures, and the speed should match that of T1 Mobile AA.

Initial balance from vanilla:

Fervor: Highest damage, slowest projectile.
Medusa: Lowest damage, fastest projectile.
Lobo: Slightly more accurate.
Zthuee: Slightly more damage.
Wavecrest: I vote for its complete removal or a change in tech level.

T1.5 units should have the same speed as artillery, equal DPS to T1 regulars, slightly more range (matching the ACU), and double the HP and cost of T1 regulars. These units will be the heavy variant of T1 land, with lower chances of killing commanders, low resistance to T1 artillery, and limited usefulness against T2. They will primarily excel in direct combat against T1 regulars. 
![image](https://github.com/user-attachments/assets/9148a002-dbd5-4291-a87b-731ddf333127)

**Comment by PurpleMossClump** [comment]
I think there's some merit to this. I suggest we let it stew, talk more about it and see about trying it out.
